syntax = "proto3";
package roboclaw_msgs;

message Move
{
    int32 left_motor = 1;
    int32 right_motor = 2;
}
